src/compiler/watch.ts

```diff
@@ -163,6 +172,10 @@ namespace ts {
             }
         }
 
+        if (reportSummary) {

```

Comment:
i agree with  @mhegazy 's https://github.com/Microsoft/TypeScript/pull/22254#issuecomment-370082170 that we should just report this all the time

Reply:
I'm hesitant to change the behavior for non-pretty non-watch users. What if those users have builds that rely on `tsc` not having any non-whitespace output? This would break their workflow.

If these changes are acceptable for watch mode, can we discuss enabling for them in a separate issue?

Reply:
But that can be done through checking ```if (program.getCompilerOptions().pretty)```